src/DacpacTool/PackageDeployer.cs

```diff
@@ -333,7 +345,12 @@ private ObjectType[] ParseObjectTypes(string value)
 
         private DacAzureDatabaseSpecification ParseDatabaseSpecification(string value)
         {
-            var specification = value.Split(";", 3);
+            if (value.Contains(';'))
```

Comment:
Mind your strings: https://www.google.com/amp/s/jimmybogard.com/mind-your-strings-with-net-5-0/amp/

Reply:
;) The certificate on that site has expired it seems.

Reply:
Use StringComparision overload

Reply:
Ah, right, yeah that makes sense :).